Networking/NetworkingTests/Mapper/ProductListMapperTests.swift

```diff
@@ -92,7 +92,7 @@ class ProductListMapperTests: XCTestCase {
 
     /// Test that ProductTypeKey converts to
     /// a ProductType enum properly.
-    func testThatProductTypeKeyConvertsToEnumProperly() {
+    func test_that_product_type_key_converts_to_enum_properly() {
```

Comment:
Maybe `test_that_productTypeKey_converts_to_enum_properly`?
```suggestion
    func test_that_productTypeKey_converts_to_enum_properly() {
```

Reply:
Good catch! Fixed in 133487a00c64f3f3ece7240be31e1c0af5084dd0.